test/integration/test_kubernetes.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from mock import patch
+
+from test import HokusaiIntegrationTestCase, TEST_KUBE_CONTEXT
+from test.utils import captured_output, mock_verbosity
+from test.mocks.mock_ecr import MockECR
+
+from hokusai import CWD
+from hokusai.lib.common import shout
+from hokusai.commands import kubernetes
+from hokusai.services.kubectl import Kubectl
+
+
+class TestKubernetes(HokusaiIntegrationTestCase):
+    @classmethod
+    def setUpClass(cls):
+        kubernetes.ECR = MockECR
+        kubernetes.HOKUSAI_CONFIG_DIR = os.path.join(CWD, 'test/fixtures/project/hokusai')
```

Comment:
@alloy these integration tests override the `CWD` constant so hokusai believes it is executing from the 'test/fixtures/project/hokusai' directory and use the Yaml fixtures there, hence the consolidation :)

Reply:
Aha. In that case I don’t really understand why in some/all cases the `HOKUSAI_CONFIG_DIR` variable still gets joined with `CWD`, e.g. [here](https://github.com/artsy/hokusai/pull/140/files/867da74da910ab7604e190e77d63cbcccbc9db33#diff-5d9ea845219ad1c2d2b2f4dd2098de7eR17). What gives?

Reply:
Ah sorry - the `HOKUSAI_CONFIG_DIR` is overridden there - the `CWD` constant is just to Dry-up calls to `os.getcwd()` across the codebase

Reply:
Gotcha, thanks!